modules/local/validatemodel/tests/nextflow.config

```diff
@@ -0,0 +1,6 @@
+nextflow.enable.moduleBinaries = true
```

Comment:
This is just fine since this is just a local module, but since I know you are trying to make them follow nf-core standards, someone recently shared with me a new convention for setting parameters for the module's nf-tests:
https://nf-co.re/docs/guidelines/components/modules#configuration-of-extargs-in-tests

You create a parameter labeled "module_args" and define it using parameter which you can also set in the nf-test main.nf.test file. You then pass this parameter to ext.args: ext.args = "params.module_args"